govyn/schemas.py

```diff
@@ -32,6 +32,11 @@ def pytype_to_schema(py_type: type) -> Dict[str, Any]:
 				'type': 'object',
 				'properties': { f.name: pytype_to_schema(f.type) for f in fields(py_type) },
 			}
+		elif isinstance(py_type, EnumMeta):
+			return {
+				'type': 'string',
```

Comment:
This might need altering in the future if we add support for int enums, but that's fine for now.